compiler/luci/pass/src/ForwardReshapeToUnaryOpPass.cpp

```diff
@@ -146,6 +148,91 @@ bool forward_reshape(luci::CircleReshape *reshape, luci::CircleLogistic *logit)
   return true;
 }
 
+bool forward_reshape(luci::CircleReshape *reshape, luci::CircleMean *mean, uint32_t axis)
+{
+  assert(reshape != nullptr); // FIX_CALLER_UNLESS
+  assert(mean != nullptr);    // FIX_CALLER_UNLESS
+
+  auto new_reshape = create_cloned_reshape(reshape);
+  if (not new_reshape)
+    return false;
+
+  // reconnect network
+  loco::replace(mean).with(new_reshape);
+  mean->input(reshape->tensor());
+  new_reshape->tensor(mean);
+
+  // Change const shape axis value
+  auto *shape_reshape = loco::must_cast<luci::CircleConst *>(new_reshape->shape());
+  assert(shape_reshape->dtype() == loco::DataType::S32);     // FIX_CALLER_UNLESS
+  assert(axis < shape_reshape->size<loco::DataType::S32>()); // FIX_CALLER_UNLESS
+  shape_reshape->at<loco::DataType::S32>(axis) = 1;
```

Comment:
```suggestion
  assert(axis >= 0); // FIX_CALLER_UNLESS
  shape_reshape->at<loco::DataType::S32>(axis) = 1;
```

Reply:
axis here is uint32_t, so this assert is unnecessary, and caller side has the same assert